diagnostic_feedback/templates/underscore/category.html

```diff
@@ -0,0 +1,13 @@
+<div class="category">
+    <input type="hidden" id="category[id][<%- order %>]" name="category[id][<%- order %>]" class="skip-validation" value="<%- id %>" />
+    <input placeholder="Name" type="text" id="category[name][<%- order %>]" name="category[name][<%- order %>]"  class="required" value="<%- name %>" />
+    <input placeholder="Image" type="text" id="category[image][<%- order %>]" name="category[image][<%- order %>]"  value="<%- image %>" />
```

Comment:
The Image field placeholder should have more of an explanation and perhaps an example. I think `placeholder="{% trans "Image URL" %} (e.g. /static/img15.jpg)"` would be much better for example.